internal/cli/auth/login.go

```diff
@@ -171,16 +122,16 @@ func (opts *LoginOpts) Run(ctx context.Context) error {
 	return nil
 }
 
-func (opts *LoginOpts) CheckProfile(ctx context.Context) error {
+func (opts *LoginOpts) checkProfile(ctx context.Context) error {
 	if err := opts.InitStore(ctx); err != nil {
 		return err
 	}
 	if config.OrgID() != "" && !opts.OrgExists(config.OrgID()) {
-		config.SetOrgID("")
+		opts.config.Set("org_id", "")
```

Comment:
can we use the config costs from [internal/config/profile.go](https://github.com/mongodb/mongodb-atlas-cli/blob/cf90d8088fefddb5f6f979d8cbfe7932993b79be/internal/config/profile.go#L49)?